app/src/main/res/layout/fragment_view_video.xml

```diff
@@ -12,7 +12,6 @@
         android:id="@+id/mediaDescription"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@+id/toolbar"
```

Comment:
lint was blocking release builds because of this warning

Reply:
Is it okay? Will it overlap?

Reply:
Oh, looks like this was never actually working. I'll fix it.